### ISSUE TITLE
Error on container boot

### ISSUE THREAD

**Issue by pixel-21** [shopwareLabs/shopware-docker#28]
I try to set up the docker environment on my MAC but alway fail on step 7.

Does anyone have an idea what could cause this error? Here is the complete error message:  

```
Step 7/35 : RUN pear config-set http_proxy $DOCKER_HTTP_PROXY
	 ---> Running in c89ed4ee87da
	config-set expects 2 or 3 parameters
	Service 'app_server' failed to build: The command '/bin/sh -c pear config-set http_proxy $DOCKER_HTTP_PROXY' returned a non-zero code: 1
```

**Comment by pixel-21** [comment]
> I created a fix with commit: [2295229](https://github.com/shopwareLabs/shopware-docker/commit/2295229e07f0b5f486e56db684f72112ef91bcb9)
> 
> Please test this fix and leave me a note if it works :-)

Thanks for the fix.

I still have the problem that I can't access the shop through http://10.100.111.46/ but this seems to be an other Mac issue.

The installation process run through without problems with the fix.

**Comment by teiling88** [comment]
Ok - so I close this issue then :-)
